canvasapi/quiz.py

```diff
@@ -50,3 +52,69 @@ def delete(self, **kwargs):
         quiz_json.update({'course_id': self.course_id})
 
         return Quiz(self._requester, quiz_json)
+
+    def get_quiz_group(self, id, **kwargs):
+        """
+        Get details of the quiz group with the given id
+
+        :calls: `GET /api/v1/courses/:course_id/quizzes/:quiz_id/groups/:id \
+        <https://canvas.instructure.com/doc/api/quiz_question_groups.html#method.quizzes/quiz_groups.show>`_
+
+
+        :param id: The ID of the question group.
+        :type id: int
+
+        :returns: `QuizGroup` object
+        :rtype: :class:`canvasapi.quiz_group.QuizGroup`
+        """
+        response = self._requester.request(
+            'GET',
+            'courses/{}/quizzes/{}/groups/{}'.format(self.course_id, self.id, id),
+            _kwargs=combine_kwargs(**kwargs)
+        )
+
+        response_json = response.json()
+        response_json.update({'course_id': self.id})
+
+        return QuizGroup(self._requester, response_json)
+
+    def create_question_group(self, quiz_groups, **kwargs):
+        """
+        Create a new question group for the given quiz id
+
+        :calls: `POST /api/v1/courses/:course_id/quizzes/:quiz_id/groups/:id \
+        <https://canvas.instructure.com/doc/api/quiz_question_groups.html#method.quizzes/quiz_groups.create>`_
+
+        :param quiz_groups: The name, pick count, question points,
+            and/or assessment question bank id.
+            All of these parameters are optional, but at least one must exist
+            (even if empty) to recieve a response.
+            The request expects a list, but will only create 1 question group per request.
+        :type quiz_groups: list[dict]
+
+        :returns: `QuizGroup` object
+        :rtype: :class:`canvasapi.quiz_group.QuizGroup`
+        """
+
+        if not isinstance(quiz_groups, list) or len(quiz_groups) <= 0:
```

Comment:
A list (`quiz_groups`) cannot contain less than one element. When a list is evaluated as a boolean (e.g. a conditional like this), an empty list evaluates to `False`.

```
>>> test = []
>>> bool(test)
False
```

So we can replace `len(quiz_groups) <= 0` with `quiz_groups`.

Reply:
Directly evaluating as a boolean is good, though I'm going to replace `len(quiz_groups) <= 0` with `not quiz_groups` instead of `quiz_groups`, as the desired result is for the expression to evaluate to `True` when the list is empty.